ldap/ldif/template-dse-minimal.ldif.in

```diff
@@ -207,6 +207,15 @@ nsslapd-pluginarg0: nsmultiplexorcredentials
 nsslapd-pluginarg1: nsds5ReplicaCredentials
 nsslapd-pluginid: des-storage-scheme
 
+dn: cn=GOST_YESCRYPT,cn=Password Storage Schemes,cn=plugins,cn=config
```

Comment:
You also need ldap/servers/slapd/fedse.c as well for "on upgrade" enabling this. Look for the PBKDF2 example there.

Reply:
Oppss I spoke too soon. Sorry!